android/src/main/java/com/bitmovin/player/reactnative/BitmovinCastManagerModule.kt

```diff
@@ -0,0 +1,74 @@
+package com.bitmovin.player.reactnative
+
+import com.bitmovin.player.casting.BitmovinCastManager
+import com.bitmovin.player.reactnative.converter.JsonConverter
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.UIManagerModule
+
+private const val MODULE_NAME = "BitmovinCastManagerModule"
+
+@ReactModule(name = MODULE_NAME)
+class BitmovinCastManagerModule(
+    private val context: ReactApplicationContext,
+) : ReactContextBaseJavaModule(context) {
+    override fun getName() = MODULE_NAME
+
+    /**
+     * Returns whether the [BitmovinCastManager] is initialized.
+     */
+    @ReactMethod
+    fun isInitialized() = BitmovinCastManager.isInitialized()
```

Comment:
Don't we need to use a `Promise` here?
Or will this be called and returned synchronously?

Reply:
good catch. [64998be](https://github.com/bitmovin/bitmovin-player-react-native/pull/254/commits/64998bef11a25e4358879f7a05c0e23ac1649b1d)